website/tests/test_website_models.py

```diff
@@ -0,0 +1,191 @@
+import pytest
+import time
+from distutils.version import LooseVersion
+from django.utils import timezone
+from ..models import Profile, WebsiteSection, NewsPost, \
+    Publication, CarouselImage, DocumentationLink
+
+
+def test_string_representation(random_user):
+    entry = WebsiteSection(title="My entry title")
+    assert str(entry) == entry.title
```

Comment:
this is also a django function right? So I think django already has the relevant tests. I am also new to testing in django so I am not sure.